routes/web.php

```diff
@@ -25,6 +25,7 @@
     Route::get('/categories/{category}/words/create', 'AdminWordsController@create');
 
 });
+Route::resource('/admin/words', 'AdminWordsController');
```

Comment:
@christanplaza 
You can group this with the admin prefix

Reply:
Fixed in 77e2cc7